py/torch_tensorrt/dynamo/conversion/_ConverterRegistry.py

```diff
@@ -81,13 +80,15 @@ class ConverterSupport:
             whether that node can be supported by its companion converter. Note that
             this function must not modify the node or its graph
         supports_dynamic_shapes: Boolean flag indicating if the converter has support for dynamic inputs.
+        requires_output_allocator: Boolean flag indicating if the converter requires to run in output allocator.
```

Comment:
`Boolean flag indicating if the converter creates operators which require a Output Allocator to run (e.g. data dependent operators)`